helix-contract/contracts/mapping-token/v2/GuardRegistry.sol

```diff
@@ -3,7 +3,7 @@
 pragma solidity >=0.8.10;
 pragma experimental ABIEncoderV2;
 
-import "../../utils/ECDSA.sol";
+import "@zeppelin-solidity-4.4.0/contracts/utils/cryptography/ECDSA.sol";
```

Comment:
In this version, `Guard` use claimId to protect signature reuse or replay protection. So it is no problem. But it's better to upgrade the newest version to prevent the potentially security issue.

Reply:
After disscussion, we use this version for now and wait for a higher version of openzipplin to be audited before doing the upgrade.